bagit-pax/pitt_pax_v2.py

```diff
@@ -17,6 +17,34 @@
 import botocore
 from concurrent.futures import ThreadPoolExecutor, as_completed
 
+#boolean to decide whether script will be automatic or not
+isAutomated = False

```

Comment:
Why omit the `global` keyword from `isAutomated`, but present it on `first_step`, et. al.?  I don't think the `global` keyword is actually doing anything on these variables outside of a function scope.

Reply:
So I had issues running the automated scripts when using the three parameters, since it would cause a variable not declared error. That's why i originally used the global descriptor for the three, but like you said I don't think I need to it might just be a scope issue for one of them?